### PR TITLE
Fixed a bug with XP not dropping

### PR DESCRIPTION
**Pull request by altaiiiir**
Tested this by mining coal. I'll be honest I got lazy and didn't track the number of XP dropped vs the expected amount, just assumed it's dropping the proper amount by default.

### CODE REVIEW

**Review by TheJaysH — COMMENTED** [review]
I made some changes to the method names to reflect their actual intent. but other than that we're all good. Happy to merge

File: src/main/java/com/ael/viner/util/MiningUtils.java [hunk 1]
```diff
@@ -49,11 +49,28 @@ public static void mineBlocks(ServerPlayer player, List<BlockPos> blocksToMine)
         for (BlockPos blockPos : blocksToMine) {
             getDrops(player, (ServerLevel) level, blockPos, tool, firstBlockPos);
 
+            getXp(level.getBlockState(blockPos), (ServerLevel) level, blockPos, tool);
```
[the remark this answers is not in the document]
Comment (altaiiiir) [reply src/main/java/com/ael/viner/util/MiningUtils.java]: I already tested this and wasn't able to get any XP drops

**Comment by altaiiiir** [comment]
Merging this

**Comment by altaiiiir** [comment]
Everything was properly tested this time including XP amounts

**Pull request merged**